backend/LexBoxApi/Controllers/ProjectController.cs

```diff
@@ -232,68 +232,6 @@ private async Task StreamHttpResponse(HttpContent hgResult)
         await hgResult.CopyToAsync(writer.AsStream());
     }
 
-    [HttpPost("updateLexEntryCount/{code}")]
-    [ProducesResponseType(StatusCodes.Status200OK)]
-    [ProducesResponseType(StatusCodes.Status404NotFound)]
-    [ProducesDefaultResponseType]
-    public async Task<ActionResult<int>> UpdateLexEntryCount(string code)
-    {
-        var result = await projectService.UpdateLexEntryCount(code);
-        return result is null ? NotFound() : result;
-    }
-
-    [HttpPost("updateLanguageList/{code}")]
-    [ProducesResponseType(StatusCodes.Status200OK)]
-    [ProducesResponseType(StatusCodes.Status404NotFound)]
-    [ProducesDefaultResponseType]
-    public async Task UpdateLanguageList(string code)
-    {
-        var projectId = await projectService.LookupProjectId(code);
-        await projectService.UpdateProjectLangTags(projectId);
-    }
-
-    [HttpPost("updateMissingLanguageList")]
-    public async Task<ActionResult<string[]>> UpdateMissingLanguageList(int limit = 10)
```

Comment:
please add this back in along with the one for missing project Id

Reply:
Done in commit cb7f2f74.